sdk/storage/azure_storage_blob/src/pipeline/storage_headers_policy.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use async_trait::async_trait;
+use azure_core::{headers::HeaderName, Context, Policy, PolicyResult, Request};
+use std::sync::Arc;
+use uuid::Uuid;
+
+#[derive(Debug, Clone)]
+pub struct StorageHeadersPolicy;
+
+#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
+#[cfg_attr(not(target_arch = "wasm32"), async_trait)]
+impl Policy for StorageHeadersPolicy {
+    async fn send(
+        &self,
+        ctx: &Context,
+        request: &mut Request,
+        next: &[Arc<dyn Policy>],
+    ) -> PolicyResult {
+        let result = request
+            .headers()
+            .get_str(&HeaderName::from("x-ms-client-request-id"));
```

Comment:
Better to define this as a `const`:

```rust
const CLIENT_REQUEST_ID: HeaderName = HeaderName::from_static("x-ms-client-request-id");
```

Though, I think we already have one that defined in `azure_core`.